src/client/theme-default/components/VPLocalSearchBox.vue

```diff
@@ -312,6 +314,16 @@ useEventListener('popstate', (event) => {
   event.preventDefault()
   emit('close')
 })
+
+/** Lock body */ 
+const isLocked = useScrollLock(document.body)
```

Comment:
directly accessing document will cause issues with ssr. make it something like

```
const body = shallowRef<HTMLElement | null>(null)

onMounted(() => {
  body.value = document.body
})

const isLocked - useScrollLock(body)
```

Reply:
ok, I will fix it.